packages/dev-server/src/createServer.ts

```diff
@@ -59,10 +65,24 @@ export async function createServer(config: Server.Config) {
     },
   });
 
+  const devMiddleware = createDevMiddleware({
+    projectRoot: config.options.rootDir,
+    serverBaseUrl: `http://${config.options.host}:${config.options.port}`,
+    // TODO verify this works properly
+    logger: instance.log,
```

Comment:
@RafikiTiki did you manage to verify whether this is ok?

Reply:
Yes, it does work flawlessly – TODO removed